src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private int radius;
+
+    public int getRadius() {
+        return radius;
+    }
+
+    public void setRadius(int radius) {
+        this.radius = radius;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: " + getName()

```

Comment:
```suggestion
        System.out.println("Figure: Circle"
```
better just put the name to the toString method (in other classes as well)